src/praisonai-agents/tests/simple_cc_test.py

```
@@ -0,0 +1,85 @@
+        os.environ.pop(key, None)
+    
+    try:
+        result = send_email(
+            to="test@example.com",
+            subject="Test",
+            body="Test body",
+            cc="cc@example.com",
+            bcc="bcc@example.com"
+        )
+        # Should not reach here
+        assert False, "Expected ValueError but got result: " + result
```

Ruff (0.15.15)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)